addons/mail/models/mail_message.py

```diff
@@ -1248,3 +1252,14 @@ def _invalidate_documents(self, model=None, res_id=None):
 
     def _get_search_domain_share(self):
         return ['&', '&', ('is_internal', '=', False), ('subtype_id', '!=', False), ('subtype_id.internal', '=', False)]
+
+    def _search_domain(self, search_term):
```

Comment:
separate method probably not necessary (and if we keep it, it probably has more value on thread for overrides rather than on message)

Reply:
Done